### PR TITLE
Getting int deploying

### PR DESCRIPTION
**Pull request by JackQuincy**
Removing all hard coding of arosvc in INT and PROD.  
passing acr resource id into rp container.

### CODE REVIEW

File: pkg/env/prod.go [hunk 1]
```diff
@@ -147,6 +151,11 @@ func (p *prod) ACRResourceID() string {
 	return os.Getenv("ACR_RESOURCE_ID")
 }
 
+func (p *prod) ACRName() string {
+	resource, _ := azure.ParseResourceID(p.ACRResourceID())
```
Comment (m1kola) [inline pkg/env/prod.go]: We already parse it in `newProd`. Can we re-use the result from `newProd`?

File: pkg/deploy/generator/resources.go [hunk 2]
```diff
@@ -779,6 +779,7 @@ ExecStart=/usr/bin/docker run \
   -e PULL_SECRET \
   -e ADMIN_API_CLIENT_CERT_COMMON_NAME \
   -e RP_MODE \
+  -e ACR_RESOURCE_ID \
```
Comment (m1kola) [inline pkg/deploy/generator/resources.go]: We've got some int values hardcoded in `pkg/env/int.go` and some variables here. Not sure that I understnad the logic behind decisions of what goes where.

File: pkg/env/prod.go (hunk 1 above)
Comment (JackQuincy) [reply pkg/env/prod.go]: good idea.  Made the change

File: pkg/env/prod.go [hunk 3]
```diff
@@ -116,6 +117,13 @@ func newProd(ctx context.Context, log *logrus.Entry, instancemetadata instanceme
 	p.clustersGenevaLoggingPrivateKey = clustersGenevaLoggingPrivateKey
 	p.clustersGenevaLoggingCertificate = clustersGenevaLoggingCertificates[0]
 
+	var acrResource azure.Resource
+	acrResource, err = azure.ParseResourceID(p.ACRResourceID())
```
Comment (nilsanderselde) [inline pkg/env/prod.go]: Can just do 
```
	acrResource, err := azure.ParseResourceID(p.ACRResourceID())
```

File: pkg/deploy/generator/resources.go (hunk 2 above)
Comment (JackQuincy) [reply pkg/deploy/generator/resources.go]: Here I'm just passing in the enviroment variable that already exists. But the idea here is in the future in prod we might need more than one ACR.

**Comment by jim-minter** [comment]
looks fine.  Am going to rebase, squash and merge.

**Pull request merged**